lib/dradis/plugins/netsparker/importer.rb

```diff
@@ -19,20 +19,35 @@ def import(params={})
       end
 
       @doc.xpath('/netsparker/vulnerability').each do |xml_vuln|
-        process_vulnerability(xml_vuln)
```

Comment:
I liked the idea of moving the responsibility of understanding how to parse the vuln data into a separate method.